Purchases/Purchasing/StoreKitAbstractions/StoreProduct.swift

```diff
@@ -47,10 +54,24 @@ public typealias SK2Product = StoreKit.Product
     /// The price of this product using ``priceFormatter``.
     @objc public var localizedPriceString: String { fatalError() }
 
+    /// The string that identifies the product to the Apple App Store.
     @objc public var productIdentifier: String { fatalError() }
+
+    /// A Boolean value that indicates whether the product is available for family sharing in App Store Connect.
+    /// Check the value of `isFamilyShareable` to learn whether an in-app purchase is sharable with the family group.
+    ///
+    /// When displaying in-app purchases in your app, indicate whether the product includes Family Sharing
+    /// to help customers make a selection that best fits their needs.
+    ///
+    /// Configure your in-app purchases to allow Family Sharing in App Store Connect.
+    /// For more information about setting up Family Sharing, see Turn-on Family Sharing for in-app purchases.
```

Comment:
`see Turn-on Family Sharing for in-app purchases` is there a URL we can point people to?

Reply:
I'll look for it 👍  I copied this from the Apple Docs. Thanks for proof reading it!

Reply:
Done